sdk/eventgrid/Microsoft.Azure.WebJobs.Extensions.EventGrid/src/EventGridExtensionConfigProvider.cs

```diff
@@ -184,6 +183,7 @@ private async Task<HttpResponseMessage> ProcessEventsAsync(JArray events, string
                 };
                 executions.Add(_listeners[functionName].Executor.TryExecuteAsync(triggerData, CancellationToken.None));
```

Comment:
Was this ever awaited previously, or just accidentally treated as fire-and-forget?

Reply:
It was synchronously awaited here - https://github.com/Azure/azure-sdk-for-net/pull/21813/files#diff-00c85e19e67ad5bd236873871ea7d7e5d5aa677cc635bf5af7d94c3848ee7b82R192